### PR TITLE
added run test

### PR DESCRIPTION
**Pull request by chris-soelistyo**
- Added run test (`tests/test_run.py`)
- Fixed image loading such that it can deal with `np.uint16` `mrc` files and convert them to `float32` torch tensors (`grace.io.image_dataset.py`)
- Added a simple extractor fixture for tests (`tests/conftest.py`) 
- Modified the random image and graph function such that the edge labels are randomised (either TP or TN) instead of UNKNOWN (`tests/_utils.py`)

### CODE REVIEW

**Comment by KristinaUlicna** [comment]
Happy to review the PRs when the checks passing gets sorted 🚀

File: setup.cfg [hunk 1]
```diff
@@ -38,7 +38,7 @@ install_requires =
   pyarrow
   seaborn
   opencv-python
-
+  click
```
Comment (KristinaUlicna) [inline setup.cfg:41]: What does this do? [sorry for my ignorance]

File: grace/io/image_dataset.py [hunk 2]
```diff
@@ -70,9 +70,7 @@ def __getitem__(self, idx: int) -> tuple[torch.Tensor, dict]:
         img_path = self.image_paths[idx]
         grace_path = self.grace_paths[idx]
 
-        image = torch.tensor(
-            self.image_reader_fn(img_path), dtype=torch.float32
-        )
+        image = torch.tensor(self.image_reader_fn(img_path).astype("float32"))
```
Comment (KristinaUlicna) [inline grace/io/image_dataset.py:73]: This works, but shouldn't the data types & loading be dealt with in the `reader` functions? In this way, we're getting the image post-processed here but potentially not in the rest of the pipeline.

**Review by KristinaUlicna — COMMENTED** [review]
Tests + setup look great, well done! 🚀 
Re: image readers, I think we need to make sure all the images are:
1. read using a defined reader
2. get post-processed equally in all steps of the pipeline. 

Not sure what's the best way to proceed, but happy to discuss.

File: setup.cfg (hunk 1 above)
Comment (chris-soelistyo) [reply setup.cfg:41]: Just a module that allows defining of hyper parameters when running files from the terminal!

File: grace/io/image_dataset.py (hunk 2 above)
Comment (chris-soelistyo) [reply grace/io/image_dataset.py:73]: I thought the idea was that the conversion wouldn't be done automatically in the rest of the pipeline? Such that we can preserve the original data types. I.e., only where it's needed (where it's out to be fed to a model) do we convert to `float32`

Comment (KristinaUlicna) [reply grace/io/image_dataset.py:73]: Ohhh now I see what you meant! I think it makes perfect sense, but should we make sure that other datatypes get process correctly here before merging in? (I'm probably just overly cautious) 👀
